src/tests/functional/inference_engine/transformations/common_optimizations/split_concat_pair_to_interpolate_fusion_test.cpp

```diff
@@ -682,3 +682,47 @@ TEST_F(TransformationTestsF, SplitConcatPairToInterpolateFusionSpatial3D2Dynamic
         function_ref = std::make_shared<ngraph::Function>(ngraph::NodeVector{ interpolate }, ngraph::ParameterVector{ input });
     }
 }
+
+TEST_F(TransformationTestsF, SplitConcatPairToInterpolateFusionSplitWithEmptyPorts) {
```

Comment:
you don't need to create the ref function in this case, it will be created automatically if it's not provided in TearDown method of TransformationTestsF class
![image](https://user-images.githubusercontent.com/21181928/193054319-6d78bb6b-b46a-45c0-b7a3-0245e627c957.png)

Reply:
Removed